pom.xml

```diff
@@ -175,17 +194,27 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-  <groupId>joda-time</groupId>
-  <artifactId>joda-time</artifactId>
-  <version>2.10.5</version>
-</dependency>
-<!-- https://mvnrepository.com/artifact/org.apache.commons/commons-exec -->
-<dependency>
-    <groupId>org.apache.commons</groupId>
-    <artifactId>commons-exec</artifactId>
-    <version>1.3</version>
-</dependency>
-
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.10.5</version>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-exec -->
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-exec</artifactId>
+      <version>1.3</version>
+    </dependency>
+    <dependency>
+      <groupId>tv.twelvetone.rjson</groupId>
```

Comment:
can you check the license usage of rjson library? Also it will create issues while releasing to Maven central when snapshots are used. please use the release versions of libraries and not the snapshot.

Reply:
i removed the rjson dependency in favor of 
`JsonParser parser = new JsonFactory()
                    .createParser(response)
                    .enable(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES)
                    .enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES);`
then operating on JsonNode to get attributes for populating Version object.